src/odemis/acq/test/flim_test.py

```diff
@@ -272,6 +279,101 @@ def test_setting_stream(self):
         # make sure there are times for each value.
         self.assertEqual(len(self._image.metadata[MD_ACQ_DATE]), self._image.shape[0])
 
+    def test_setting_stream_rep(self):
+        """
+        Check the ROI <> Repetition <> pixelSize connections work fine
+        """
+
+        helper = stream.ScannedTCSettingsStream('Stream', self.apd, self.ex_light, self.lscanner,
+                                                self.sft, self.tc_scanner)
+
+        # Ask full ROI -> should have full ROI
```

Comment:
How do you ask for full ROI? Setting the square selection?

Reply:
Yes, that's how the ROI is defined. Full ROI = (0,0,1,1).